Storage/container/many_to_many.h

```diff
@@ -105,12 +123,27 @@ class GenericManyToMany {
     return operator[](std::make_pair(row, col));
   }
 
+  ImmutableOptional<std::chrono::microseconds> LastModified(const key_t& key) const {
```

Comment:
Where do you want to see it?
It's self-explanatory by definition :)
`using key_t = std::pair<row_t, col_t>;`